build.gradle

```diff
@@ -58,7 +65,7 @@ compileTestKotlin {
 }
 
 markdownToHtml {
-    sourceDir file("$projectDir/docs")
+    sourceDir file("${project.buildDir}/markdown")
```

Comment:
Things being put in this directory:

- The plugin description gets copied into this directory from the `docs` dir.
- The plugin `RELEASE-NOTES.md` gets built from an environment variable and gets written to this directory as well.